x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -166,9 +166,10 @@ public static void main(String[] args) throws Exception {
             "      disable hostname verification in your SSL configuration.";
 
     static final String CA_EXPLANATION =
-        "    * All certificates generated by this tool will be signed by a certificate authority (CA).\n" +
-            "    * The tool can automatically generate a new CA for you, or you can provide your own with the\n" +
-            "         -ca or -ca-cert command line options.";
+        "    * All certificates generated by this tool will be signed by a certificate authority (CA)\n" +
+            "      unless the --self-signed command line option is specified.\n" +
+            "      The tool can automatically generate a new CA for you, or you can provide your own with " +
+            "      the -ca or -ca-cert command line options.";
```

Comment:
Should these have two dashes?
```suggestion
            "      the --ca or --ca-cert command line options.";
```

Reply:
We haven't really been very consistent on the usage of single dash and double dashes. There are still many other places that use single dash. For this particular paragraph, I am ok to make it all double dashes.